corehq/apps/domain/views/settings.py

```diff
@@ -271,6 +273,71 @@ def post(self, request, *args, **kwargs):
         return self.get(request, *args, **kwargs)
 
 
+class EditIPAccessConfigView(BaseProjectSettingsView):
+    template_name = 'domain/admin/ip_access_config.html'
+    urlname = 'ip_access_config'
+    page_title = gettext_lazy("IP Access")
+
+    @method_decorator(always_allow_project_access)
+    @method_decorator(login_and_domain_required)
+    @use_bootstrap5
+    def dispatch(self, *args, **kwargs):
+        if IP_ACCESS_CONTROLS.enabled(self.domain):
+            return super().dispatch(*args, **kwargs)
+        else:
+            raise Http404()
+
+    @property
+    @memoized
+    def form(self):
+        initial = {}
+        domain_config = self.get_ip_access_config
+        if domain_config:
+            display_spacer = ", "
+            initial.update({
+                'country_allowlist': domain_config.country_allowlist,
+                'ip_allowlist': display_spacer.join(domain_config.ip_allowlist),
+                'ip_denylist': display_spacer.join(domain_config.ip_denylist),
+                'comment': domain_config.comment
+            })
+
+        if self.request.method == 'POST':
+            return IPAccessConfigForm(self.request.POST, initial=initial)
+        return IPAccessConfigForm(initial=initial)
+
+    @property
+    def get_ip_access_config(self):
+        try:
+            return IPAccessConfig.objects.get(domain=self.domain)
+        except IPAccessConfig.DoesNotExist:
+            return None
+
+    @property
+    def page_context(self):
+        return {
+            'ip_access_config_form': self.form,
+        }
+
+    def post(self, request, *args, **kwargs):
+        if self.form.is_valid():
+            domain_config = self.get_ip_access_config
+            should_save = True
+            if not domain_config:
+                should_save = False
+                domain_config = IPAccessConfig()
+                domain_config.domain = self.domain
+
+            for attr, value in self.form.cleaned_data.items():
+                if value:
```

Comment:
```suggestion
                if value != getattr(domain_config, attr):
```
Without this, the user won't be able to blank out any fields.  For instance, if they have a comment on the saved `domain_config`, then submit the form to delete that comment, `value` will be falsy and this won't update.